Shu/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.10-slim-bullseye
+COPY requirements.txt /home
```

Comment:
```suggestion
```

Your current `requirements.txt` isn't used at all for the runtime of the container, and Docker containers shouldn't have additional files in them that aren't required for runtime or later use. So this line can be removed.